lib/aot/compiler.cpp

```diff
@@ -5146,6 +5149,9 @@ Expect<void> Compiler::compile(Span<const Byte> Data, const AST::Module &Module,
     if (!Conf.getCompilerConfigure().isGenericBinary()) {
       CPUName = llvm::sys::getHostCPUName();
     }
+#if defined(__riscv) && __riscv_xlen == 64
+    CPUName = "generic-rv64";
```

Comment:
Just want to know if `llvm::sys::getHostCPUName()` will return `generic-rv64` or not.
I think this should be moved to line 5145. And you can create a constant string `"generic"` or `"generic-rv64`, depends on the platform.

Reply:
In my environment `llvm::sys::getHostCPUName()` did not return `generic-rv64`, so I defined it. I would move it to line 5145 as you said.